### DISCUSSION TITLE
why is this running so slow.

### DISCUSSION THREAD

**Discussion by elen07zz** [vladmandic/sdnext#731] [General]
i dont know why, but im getting way worse it/s 

i have tried disabling lycoris.
But its not working either.

**Comment by vladmandic** [comment]
way worse it/s than what? what is your platform? what were you getting before? what were you using when you were getting those better numbers? which version are you using now? can you run system info benchmark and post results?
its nearly impossible to help if there is zero information provided.

**Reply by elen07zz** [reply]
worse than like a week before. im using amd rx 6700. I'm getting like maybe 1/4 speed.
Using the latest version.

**Reply by vladmandic** [reply]
very likely same as #732, still trying to figure that one out.

**Reply by elen07zz** [reply]
sounds similar, yes.. even using 512x512 and low stesps (10) im getting similar speeds as he said..  mm i dont have a game installed right now to taste what he said. Im gonna try installing one later and see if that works..
As he, other UIs works fine.

**Reply by OldPixelReaper** [reply]
For some unknown reason, after a git pull, it only runs on the CPU for me, which is why it's so slow. 48 hours ago everything was still working. Reinstallation has also brought nothing.

Win11, AMD RX 5700 XT

**Reply by iDeNoh** [reply]
I believe that's what was happening to me, it was saying my GPU was being used but I was getting CPU speeds

**Reply by kotojichang** [reply]
I'm having similar issue, somehow it was using my CPU instead of GPU (AMD RX 6800 XT), it was fine yesterday until I git pull, reinstall rollback all does not help either. I tried run on https://github.com/lshqqytiger/stable-diffusion-webui-directml it was works fine.

**Comment by vladmandic** [comment]
@kotojichang see #732 - best guess so far is that seems like there is an issue in **ROCm** which causes it to run in the wrong power state until its "nudged".

**Reply by kotojichang** [reply]
Sry for the late reply, I think I didn't make it clear, I run in Windows 10 so shouldn't have ROCm support build in yet right? From my system monitoring I can see it using my CPU when run the generation, my GPU just idle.

**Reply by vladmandic** [reply]
@kotojichang that's correct - amd did not release rocm for windows yet. i replied to you, but meant for others that report their amd gpu performance (when using rocm) varies depending on phase of the moon (you reboot and its slow, you play a game for a min and return to this app and its fast - because something needs to "nudge" gpu to a high-performance gpu state and rocm by itself cannot do that).

**Reply by elen07zz** [reply]
only with this app for some reason automatic1111 is working just fine.

**Reply by vladmandic** [reply]
@elen07zz that's also proven to be incorrect as some users have switched to a1111 and have same issues and some dont. especially since a1111 also upgraded to torch 2.0 and newer version of rocm.

**Reply by elen07zz** [reply]
> @elen07zz that's also proven to be incorrect as some users have switched to a1111 and have same issues and some dont. especially since a1111 also upgraded to torch 2.0 and newer version of rocm.

ok, well at least in my case that's how it is.
Im using rocm 5.4
with the latest pytorch i even tried the nightly version.
